### DISCUSSION TITLE
Updating Login/Digest Email

### DISCUSSION THREAD

**Discussion by writingpoli** [knowledgefutures/pubpub#1900] [Support] [answered]
Where can we update our login email? Similarly, where can we access/change our digest subscription email? 

I'm assuming mine is going to my dead login email. 

Thank you!

**Comment by gabestein** [comment, marked as answer]
Apologies on the delay here. I just followed up with you via email since this is a bit sensitive. But for future users' benefit, the digest email gets sent to your account email. If you're not seeing it, check to make sure it hasn't gone there, and add hello@mg.pubpub.org to your contacts.

**Reply by shaunagm** [reply]
This is marked as answered, but I don't think it answers the question "how do I change my login email" which a colleague of mine now has. Do you have any guidance for us? Thank you. :)

**Reply by gabestein** [reply]
Sorry about that. You can have your colleague email help@pubpub.org to change the email address. Please have them write from the existing email so we can confirm account ownership.
